nano/node/bootstrap/bootstrap_lazy.hpp

```diff
@@ -0,0 +1,113 @@
+#pragma once
+
+#include <nano/node/bootstrap/bootstrap_attempt.hpp>
+#include <nano/node/bootstrap/bootstrap_bulk_pull.hpp>
+#include <nano/node/common.hpp>
+#include <nano/node/socket.hpp>
+#include <nano/secure/blockstore.hpp>
+#include <nano/secure/ledger.hpp>
+
+#include <boost/multi_index/hashed_index.hpp>
+#include <boost/multi_index/member.hpp>
+#include <boost/multi_index/ordered_index.hpp>
+#include <boost/multi_index_container.hpp>
+#include <boost/thread/thread.hpp>
+
+#include <atomic>
+#include <future>
+#include <queue>
+#include <unordered_set>
+
+namespace mi = boost::multi_index;
+
+namespace nano
+{
+class bootstrap_attempt;
+class bootstrap_client;
+class node;
+namespace transport
+{
+	class channel_tcp;
+}
```

Comment:
`bootstrap_attempt`, `bootstrap_client` & `channel_tcp` seem unused in this file

Reply:
Updated